docs/docs/marathon-ui.md

```diff
@@ -66,6 +66,53 @@ True if the `queue` endpoint returns the
 following JSON conditions:
 `queueEntry.delay.overdue === false && queueEntry.delay.timeLeftSeconds > 0`
 
+## Application Health Reference
+It is possible to specify health checks to be run against an application's
+tasks. For instructions on how to set up and use health checks, please refer to the
+[Health Checks page]({{ site.baseurl }}/health-checks.html) page.
+
+<img src="{{ site.baseurl }}/img/marathon-ui-health.png" width="700" alt="Health States in Marathon UI">
+
+An application's task lifecycle always falls into one of the following
+conditions.
+
+#### Healthy
+A task is considered Healthy when all of the supplied Health Checks are
+passing.
+
+-----------
+
+#### Unhealthy
+A task is considered Unhealthy whenever one or more health checks are reported
+as failing.
+
+-----------
+
+#### Staged
+A task is Staged when a launch request has been submitted to the cluster.
```

Comment:
...but has not been reported as running yet. Fetching the specified "uris" in the app definition or the docker images happens before Mesos reports the task as running.